.github/workflows/create-issue-with-branch.yml

```
@@ -29,16 +29,27 @@ jobs:
+          script: |
+            const body = context.payload.issue.body;
+            const keywordMatch = body.match(/### 🔑 Keyword\s*\n+([^\n]+)/);
+            let keyword = keywordMatch ? keywordMatch[1].trim() : null;
+            if (keyword === null) {
+                keyword = `issue-${context.payload.issue.number}`;
+            } else {
+                keyword = keyword.toLowerCase();
+                keyword = keyword.replace(/[^a-z0-9-]+/g, '-');
+                keyword = keyword.replace(/-{2,}/g, '-');
+                keyword = keyword.replace(/^-+|-+$/g, '');
+                
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)